custom_components/localtuya/sensor.py

```diff
@@ -111,3 +87,5 @@ def unit_of_measurement(self):
     def status_updated(self):
         """Device status was updated."""
         self._state = self.dps(self._dps_id)
+        if isinstance(self._state, float):
+            self._state = round(self._state, DEFAULT_PRECISION)
```

Comment:
This doesn't truncate the numbers, so I still get values like 220.70000000000002 V....

Reply:
It's supposed to round to two digits. Is the rounding performed at all (is the `if` taken)?

Reply:
I fixed the rounding now!